pkg/plugin/oauth2/middleware_access_rules_test.go

```diff
@@ -15,174 +15,129 @@ import (
 
 const signingAlg = "HS256"
 
-func TestBlockJWTByCountry(t *testing.T) {
-	secret := "secret"
+func generateToken(alg, key string) (string, error) {
+	token := basejwt.NewWithClaims(basejwt.GetSigningMethod(alg), basejwt.MapClaims{
+		"country":  "de",
+		"username": "test@hellofresh.com",
```

Comment:
Mocked info?

Reply:
Yes, those are mocked / static claims for testing the access rules. (They were previously in between test cases, I moved them to the top.)